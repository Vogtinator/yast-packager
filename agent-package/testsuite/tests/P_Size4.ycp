{
return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"german","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Execute(.setInstallSelection,[
"aaa_base",
"aaa_dir",
"aaa_skel",
"base",
"bash",
"bc",
"bdflush",
"cpio",
"cracklib",
"cron",
"devs",
"diff",
"ext2fs",
"file",
"fileutil",
"find",
"gawk",
"gdbm",
"gppshare",
"groff",
"gzip",
"kbd",
"kernmod",
"ldso",
"libz",
"lilo",
"man",
"mktemp",
"modules",
"ncurses",
"net_tool",
"netcfg",
"nkita",
"nkitb",
"pam",
"perl",
"ps",
"rpm",
"sh_utils",
"shadow",
"shlibs",
"syslogd",
"sysvinit",
"tcsh",
"termcap",
"textutil",
"util",
"yast"
    ],false );
}
{
    return SCR::Read(.installSet);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":1000000]]);
}
{
    return SCR::Read(.neededCDs);
}

