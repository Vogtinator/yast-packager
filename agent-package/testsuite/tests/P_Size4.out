(true)
(false)
([["a1/aaa_base.rpm", "SuSE Linux Basispaket", 1, true], ["a1/aaa_dir.rpm", "SuSE Linux Verzeichnisstruktur", 1, true], ["a1/aaa_skel.rpm", "SuSE Linux Default-Benutzer", 1, false], ["a1/base.rpm", "Einige wichtige GNU-Programme", 1, true], ["a1/bash.rpm", "Die GNU Bourne-Again Shell", 1, true], ["a1/bc.rpm", "GNU bc - Kommandozeilen-Rechner", 1, false], ["a1/bdflush.rpm", "Linux-Update-Daemon", 1, true], ["a1/bzip.rpm", "Programm zur Dateikompression", 1, false], ["a1/cpio.rpm", "GNU cpio Backup- und Archiv-Utilities", 1, true], ["a1/cracklib.rpm", "Library, um Passwörter zu checken", 1, true], ["a1/cron.rpm", "Cron-Daemon", 1, true], ["a1/db.rpm", "Berkeley DB Datenbank Bibliothek", 1, true], ["a1/devs.rpm", "Gerätedateien", 1, true], ["a1/file.rpm", "Ermitteln des Dateityps", 1, true], ["a1/gawk.rpm", "GNU gawk", 1, true], ["a1/gdbm.rpm", "GNU Datenbankfunktionen", 1, true], ["a1/glibc.rpm", "Die shared Libraries der C-Library", 1, true], ["a1/gppshare.rpm", "Dynamische C++-Bibliotheken", 1, true], ["a1/groff.rpm", "GNU troff Textformatierungssystem", 1, true], ["a1/gzip.rpm", "GNU zip Komprimierungs-Utilities", 1, true], ["a1/kbd.rpm", "Tastaturtabellen, Consolefonts", 1, true], ["a1/libz.rpm", "Kompressions-Bibliothek", 1, false], ["a1/lilo.rpm", "LILO - Der Bootloader", 1, true], ["a1/man.rpm", "Programm zum Anzeigen von Manpages", 1, true], ["a1/mktemp.rpm", "Utility für temporäre Dateien", 1, true], ["a1/ncurses.rpm", "Neue Curses-Bibliotheken", 1, true], ["a1/netcfg.rpm", "Netzwerk-Konfigurationsdateien in /etc", 1, true], ["a1/pam.rpm", "PAM (Pluggable Authentication Modules)", 1, true], ["a1/perl.rpm", "Perl-Interpreter", 1, true], ["a1/ps.rpm", "Prozessinformationen", 1, true], ["a1/readline.rpm", "Die readline Bibliothek", 1, true], ["a1/rpm.rpm", "RPM Package Manager", 1, true], ["a1/shadow.rpm", "Shadow-Passwort-System", 1, true], ["a1/syslogd.rpm", "Syslogd", 1, true], ["a1/sysvinit.rpm", "SysV-Style init", 1, true], ["a1/textutils.rpm", "GNU textutils", 1, true], ["a1/yast.rpm", "Yet another Setup Tool", 1, true], ["ap1/gfxboot.rpm", "Grafisches Bootlogo für lilo und syslinux.", 1, false], ["ap1/tcsh.rpm", "Die tcsh", 1, false], ["d1/popt.rpm", "Bibliothek zum Parsen von Kommandozeilen", 1, false], ["a2/termcap.rpm", "Die libtermcap", 2, false]])
([$["free":886470, "name":"/", "used":113530]])
([100005, 187, 0, 0, 0, 0, 0])
